src/Medidata.ZipkinTracer.Core/ZipkinClient.cs

```diff
@@ -159,18 +156,12 @@ private string GetSpanName(Uri uri, string methodName)
 
         private bool IsConfigValuesValid(IZipkinConfig zipkinConfig)
         {
-            if (String.IsNullOrEmpty(zipkinConfig.ZipkinServerName))
+            if (String.IsNullOrEmpty(zipkinConfig.ZipkinBaseUri))
             {
                 logger.Error("zipkinConfig.ZipkinServerName is null");
```

Comment:
The config entry is renamed from ZipkinServerName to ZipkinBaseUri so I recommend to update the log message accordingly.